COMPILING.md

```diff
@@ -186,3 +186,20 @@ If you want to build the installer, please run the `deploy_mac.sh` script: `./ma
   `git submodule update --init`
 - Open Jamulus.pro in Qt Creator
 - Now you should be able to Build & Run for Android.
+
+## Compile time arguments
+
+During compile time some CONFIG arguments can be given to enable or disable specific features. Just run `qmake "CONFIG+=<insert build time args>"`. The following table shows available compile time options:
+
+| Option                  | Description                                                             |
+| ----------------------- | ----------------------------------------------------------------------- |
+| `serveronly`            | Only support running as Server                                          |
+| `headless`              | Disable GUI. Supports Client and Server. Usually used with serveronly   |
+| `nojsonrpc`             | Disable JSON-RPC support                                                |
+| `jackonwindows`         | Use JACK instead of ASIO on Windows                                     |
+| `jackonmac`             | Use JACK instead of CoreAudio on macOS (untested)                       |
+| `server_bundle`         | macOS only: Create an application bundle which starts server by default |
+| `opus_shared_lib`       | Use external OPUS library                                               |
+| `disable_version_check` | Disable message shown when an update is available                       |
```

Comment:
> Disable message shown when an update is available

Does it only do that?  Or does it disable _checking_ to see if a new version exists (and hence won't issue the message)?

Reply:
Actually: https://github.com/jamulussoftware/jamulus/blob/master/src/clientdlg.cpp#L809-L823
so it sees as if it doesn't disable the message but actually skip the version check

Reply:
So?
```suggestion
| `disable_version_check` | Skip checks for version updates                                         |
```